src/Symfony/Bundle/SecurityBundle/ValueResolver/SecurityUserValueResolver.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\SecurityBundle\ValueResolver;
```

Comment:
Got my vote but I don't know if it's a good idea to put things directly in the bundle root